File: 백준/Bronze/1924. 2007년/2007년.py
import sys

day_of_week = {
    0: 'MON',
    1: 'TUE',
    2: 'WED',
    3: 'THU',
    4: 'FRI',
    5: 'SAT',
    6: 'SUN'
}
months = (0, 31, 28, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31)
x, y = map(int, sys.stdin.readline().split())
day = 0
while x > 1:
    x -= 1
    day += months[x]
day += y - 1
print(day_of_week[day % 7])

File: 백준/Bronze/1924. 2007년/2007년.cc
#include <iostream>
#include <string>
using namespace std;

int main() {
	cin.tie(0)->sync_with_stdio(0);

	int month, date;
	cin >> month >> date;
	int daysInMonth[12] = { 31,28,31,30,31,30,31,31,30,31,30,31 };
	string days[7] = { "SUN", "MON", "TUE", "WED", "THU", "FRI", "SAT" };
	int totalDays = 0;
	totalDays += date;
	for (int i = 0; i < (month - 1); i++) {
		totalDays += daysInMonth[i];
	}
	cout << days[totalDays % 7];
}